README.md

```diff
@@ -47,6 +47,27 @@ if __name__ == "__main__":
 ### Variable naming
 * Generally, choose nouns for variables and verbs for methods
 * Clear variable and method names can reduce the need for comments
+* Carefully determine whether temporary variables are helpful. If you only use
+  it once, consider if that line of code can be combined with the place it is
+  used.
+* Avoid **Magic Numbers** - numerical constants without a clear purpose
+    * provide numerical constants with a variable to describe their purpose
+    * these can be physical constants (`avogadro = 6.02e23`), unit conversions
+      (`ev2Mev = 1e-6`), or vector indices (`z = 2`), among others
```

Comment:
I think `avogadro` and `ev2mev` should be all caps since they are pseudo-constants.

Reply:
I'm less fussed about that than I am about the magic number part....